library/src/main/java/org/xmtp/android/library/ApiClient.kt

```diff
@@ -16,20 +16,22 @@ import org.xmtp.proto.message.api.v1.MessageApiOuterClass.PublishResponse
 import org.xmtp.proto.message.api.v1.MessageApiOuterClass.QueryRequest
 import org.xmtp.proto.message.api.v1.MessageApiOuterClass.QueryResponse
 import java.io.Closeable
+import kotlinx.coroutines.flow.Flow
 import java.util.concurrent.TimeUnit
 
 interface ApiClient {
     val environment: XMTPEnvironment
     fun setAuthToken(token: String)
-    suspend fun queryStrings(
+    suspend fun query(
         topics: List<String>,
         pagination: Pagination? = null,
         cursor: Cursor? = null,
     ): QueryResponse
 
-    suspend fun query(topics: List<Topic>, pagination: Pagination? = null): QueryResponse
+    suspend fun queryTopic(topics: List<Topic>, pagination: Pagination? = null): QueryResponse
```

Comment:
I felt like query and queryTopic felt more clear than query and queryStrings since queryStrings is just a list of topic descriptions.

Reply:
I renamed this one last time to `queryTopics` since it's multiple topics. 😮‍💨